src/vlasovsolver/vlasovmover.cpp

```diff
@@ -22,77 +22,71 @@
 
 #include <cstdlib>
 #include <iostream>
-#include <vector>
 #include <stdint.h>
+#include <vector>
```

Comment:
The whole file seems to have only white-space and indentation diffs, I probably missed whatever was significant. Also, quite a few related to having the curly brackets after instead of before the line break are against the style of the rest of the code.

Reply:
I leave it up to @ursg about the changes in e.g. indentation and curly bracket style, but if it's not done in the whole code at once and with everyone knowing about it, the code will become a lot more difficult to read if there's different styles in different places in the code. Also, such massive white-space changes are rather done in a separate pass to make sure there's no hidden bug in the middle of thousands of lines of stuff.

If/when we merge the configure/repo sorting PR this will become a lot easier to read.

My four comments above only pertain to the last commit as I was looking only at those diffs by mistake.

As mentioned orally, the reason why I dragged throughout "sysboundary" instead of just "boundary" when the system boundaries were coded at the time is because there was places in the code where a clear distinction between system and process boundaries is (was?) needed. If this is no longer seen as a potential pitfall I don't mind if it gets changed to just "boundary".

Reply:
I was worried about some of these style changes. I used some auto-styling tools to help formatting the source code, so it will potentially generate many differences like this. Sometimes I just couldn't control myself clicking the tool buttons...

But I also think instead of manually going over each line that changes, we need to have some auto CI checks and tests that can help us quickly identifying if anything changes for the existing verification tests. Urs mentioned to me there is one written in shell script that can run tests and compare with another git version. It might be helpful to introduce some easy-to-use standard tests with known results already in the repo and run it with, e.g., `make test flowthrough`.

Reply:
I think we need to automatise that indeed quite soon.

Reply:
I guess it's time we produce some preset files for the common editors (Vim? Emacs? VS Code?), so that indentation comes out right for everybody. Also, I guess an automatic a preset for an automatic code formatter like ClangFormat might be good.
I made a [separate Issue](https://github.com/fmihpc/vlasiator/issues/530) out of this.

If your commits already make these files more conforming to the [Vlasiator Coding Style](https://github.com/fmihpc/vlasiator/wiki/Coding-Style), then that's great. I wonder if we should pull that in as a separate pull request first though, just so that things are a little less confusing.

Reply:
My latest commits followed the coding style, except some places where there might be missing curly braces around a single if statement.